static/css/less/breakpoints.less

```diff
@@ -1,2 +1,3 @@
+@width-breakpoint-mobile: 480px; // 30em
```

Comment:
This is a bit too high for mobile.

I left out a mobile breakpoint intentionally here as it forces mobile first development for anything up to 768px. Personally I'd like to avoid adding a mobile breakpoint as my experience has shown it will be abused.